lib/utils/bower-helpers.js

```diff
@@ -32,11 +32,13 @@ module.exports = {
   },
   resetBowerFile: function(root){
     var copy = RSVP.denodeify(fs.copy);
-    return copy('bower.json.ember-try', 'bower.json', {cwd: root});
```

Comment:
It doesn't appear that `cwd` is honored. I think that this was working by happenstance, since `process.cwd()` and `root` are typically identical